src/Documentation/doc/advanced/corporation/faq.md

```diff
@@ -119,33 +119,33 @@ Yes. In fact, you must do that for maximum efficiency. Check this [section](./bo
 
 They are the materials that boost [division production multiplier](./boost-material.md). There are 4 boost materials: AI Cores, Hardware, Real Estate, and Robots.
 
-#### How many boost materials that I should buy?
+#### How many boost materials should I buy?
 
 Check the optimizer in this [section](./boost-material.md).
 
-#### Why does [General advice](./general-advice.md) section tell me to use API to upgrade office size?
+#### Why does the [General advice](./general-advice.md) section tell me to use API to upgrade office size?
 
 API (`upgradeOfficeSize`) gives you granular control over office size. You cannot do that through UI.
 
 #### Why does setAutoJobAssignment not take effect immediately?
 
-It only takes effect in next cycle's START state.
+It only takes effect in the next cycle's START state.
 
-#### Why does energy and morale matter?
+#### Why do energy and morale matter?
```

Comment:
Comment: I'm pretty sure "do" is the correct one here, but "Why does energy and morale matter?" doesn't sound wrong, either. A subtle point with mass-nouns, curious.